docs/docker-files.md

```diff
@@ -0,0 +1,18 @@
+Docker Files
+============
+
+1. nginx-proxy: EasyEngine nginx-proxy uses [jwilder/nginx-proxy](https://github.com/jwilder/nginx-proxy) as the base. Nginx-proxy sets up a container running nginx and docker-gen. docker-gen generates reverse proxy configs for nginx and reloads nginx when containers are started and stopped. The root permission in this image have been changed with user permission in the EasyEngine image.
```

Comment:
Traefik instead of nginx-proxy. Also, default traefik as of now. Will update if we modify.